config/find-eu-exit-guidance-business.yml

```diff
@@ -0,0 +1,566 @@
+---
+base_path: "/find-eu-exit-guidance-business"
+content_id: 42ce66de-04f3-4192-bf31-8394538e0734
+signup_content_id: 2818d67a-029a-4899-a438-a543d5c6a20d
+title: Find EU Exit guidance for your business
+description: ''
+name: 'Find EU Exit guidance for your business'
+locale: en
+public_updated_at: '2018-11-23T09:00:00.000+00:00'
+publishing_app: specialist-publisher
+rendering_app: finder-frontend
+details:
+  beta: false
+  document_noun: publication
+  subscription_list_title_prefix: 'Find EU Exit guidance for your business '
+  reject:
+    sector: "_MISSING"
```

Comment:
Should be `sector_business_area`

Also, we should check that this works before merging.